blocks/library/quote/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query as hpq } from 'api';
+import Editable from 'components/editable';
+
+const { html, query } = hpq;
+
+registerBlock( 'core/quote', {
+	title: wp.i18n.__( 'Quote' ),
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: ( node ) => query( 'blockquote > p', html() )( node ).join(),
```

Comment:
> Is this "can" as in: technically possible per spec, or something we want to support?

I don't know :P. What I know is that TinyMCE (current implementation) adds `p` automatically around text (we could avoid it passing a tagName, but we create other limits by doing so). Also the current post content sample contains the `p`